test/e2e/testdata/es-durable-consumer.yaml

```diff
@@ -3,6 +3,8 @@ kind: EventSource
 metadata:
   name: e2e-durable-consumer
 spec:
+  template:
+    serviceAccountName: argo-events-sa
```

Comment:
Added the `argo-events-sa` service account to tests as the kafka eventsource requires a kubernetes lease and the default service account does not have permission to create these objects in the ci tests